### DISCUSSION TITLE
Connecting bluetooth OBD reader to Renault Twingo III

### DISCUSSION THREAD

**Discussion by TwingoIII** [cedricp/ddt4all#1158] [Q&A] [answered]
Hi! Can someone tell me how can I connect my obd reader to my car?
I have problem to connect to the program.
Initilizing ELM with speed 115200...
Trying to open port COM5 @ 115200
Serial<id=0x2337b9416c0, open=True>(port='COM5', baudrate=115200, bytesize=8, parity='N', stopbits=1, timeout=5, xonxoff=False, rtscts=False, dsrdtr=False)
Trying to open port COM5 @ 38400
Error: could not open port 'COM5': PermissionError(13, 'Käyttö estetty.', None, 5)
ELM not connected or wrong COM port COM5
Trying to open port COM5 @ 115200
Serial<id=0x2337b9413f0, open=True>(port='COM5', baudrate=115200, bytesize=8, parity='N', stopbits=1, timeout=5, xonxoff=False, rtscts=False, dsrdtr=False)
4866 loaded ECUs in database.
js: Uncaught SyntaxError: Failed to execute 'matches' on 'Element': ':modal' is not a valid selector.
js: Uncaught SyntaxError: Failed to execute 'matches' on 'Element': ':modal' is not a valid selector.
js: Uncaught SyntaxError: Failed to execute 'matches' on 'Element': ':modal' is not a valid selector.

Is someone familiar with this problem?

**Comment by AndyIreland** [comment, marked as answer]
hi - try downloading this version of DDT4ALL from here and you can try see if that works, its older version but seems stable. 
https://www.drive2.ru/b/498093336985338243/
Choose 'ALL CARS'
it has the ECU zip built in and puts a shortcut on your desktop
Good luck.

**Reply by TwingoIII** [reply]
Okay I will try! Thank You!
